### ISSUE TITLE
Unable to run rosrun ardrone_autonomy ardrone_driver

### ISSUE THREAD

**Issue by ekelleyv** [AutonomyLab/ardrone_autonomy#12]
Please forgive me if this is the wrong place to ask this. I am working with the ardrone for my senior thesis and ardrone_autonomy looks absolutely awesome. Thanks for your hard work.

I was wondering if you could check out the issue I am having. When I try and run rosrun ardrone_autonomy ardrone_driver, I get the following output, ending in "The AR-Drone has a suspicious Firmware number." 

https://gist.github.com/3825341

As far as stats, I am running Ubuntu 12.04 and have an ardrone 2. I would appreciate any help you could offer. Thanks!

**Comment by mani-monaj** [comment]
> Please forgive me if this is the wrong place to ask this. I am working with the ardrone for my senior thesis and ardrone_autonomy looks absolutely awesome. Thanks for your hard work.

Your are welcome. This is the best place to ask such questions :)

> I was wondering if you could check out the issue I am having. When I try and run rosrun ardrone_autonomy ardrone_driver, I get the following output, ending in "The AR-Drone has a suspicious Firmware number."
> 
> https://gist.github.com/3825341
> 
> As far as stats, I am running Ubuntu 12.04 and have an ardrone 2. I would appreciate any help you could offer. Thanks!

The driver will show you a **warning** whenever for any reason the firmware number it obtains from the Drone is invalid (in your case 0.0.0). This will **not** stop the driver from working. 

Does this happen every time you connect to the Drone?

**Comment by ekelleyv** [comment]
This was happening every time I connected to the drone. I guess what stood out to me was the statements:

VP_Com : Failed to open socket for navdata
and
FTOSFC: No route to host

All of my ubuntu network settings should be correct as I was able to run the ardrone_navigation program contained in the normal SDK 2.0. Is this something to do with how I have ROS set up? I have never used ROS before, so maybe some of my setup was wrong. Thanks for your help!
